examples/Videos/Sample/ViewController.m

```diff
@@ -55,16 +56,6 @@ - (void)viewWillAppear:(BOOL)animated
   [self.view addSubnode:_rootNode];
 }
 
-- (void)viewDidLayoutSubviews
```

Comment:
Being able to delete this is really fantastic!

However, instead of setting the frame in the appearance method, should we instead set it in the viewWillLayoutSubviews method? This may be an important distinction for rotation.